Sources/Core/Sources/Client.swift

```diff
@@ -14,7 +14,7 @@ public final class Client: @unchecked Sendable {
   /// The game this client is playing in.
   public var game: Game
   /// The client's configuration
-  public var configuration = ClientConfiguration() {
+  public var configuration: ClientConfiguration {
     didSet {
```

Comment:
I think this was probably broken before your changes were made (by changing the configuration type from a struct to a class), but I think this doesn't ever get triggered unless the configuration object gets set to a new object entirely (which never happens). The new ClientConfiguration protocol will have to have some way for the client to listen for changes to the configuration. Since this was already broken I'm happy for you to just create a new issue for this and we can solve that another time.

Also, change this to a `let` now since there's no syncing between Client and PlayerInputSystem/Game, so they would end up talking to different configuration objects if someone set this property manually.

Reply:
I've removed the whole closure for now and changed it to a let constant